svcdiscovery/resolvwriter/main.go

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
While it could be a 3-line shell script, it's on an image that has no shell! :D
I was even writing this in C but felt too complicated for other people to do maintenance in the future.
Regarding where it lives, I'm fine putting it wherever with its own description.